compiler/commands.nim

```diff
@@ -87,10 +87,11 @@ proc writeFullhelp(conf: ConfigRef; pass: TCmdLinePass) =
 
 proc getNimSourceData(): tuple[hash, date: string] {.compileTime.} =
   ## Retrieve metadata about the compiler source code.
-  let hashCall = gorgeEx("git rev-parse --verify HEAD")
-  let dateCall = gorgeEx("git log -1 --format=%cs HEAD")
-  if hashCall.exitCode == 0 and dateCall.exitCode == 0:
-    result = (hashCall.output.strip(), dateCall.output.strip())
+  const
```

Comment:
Things like these really make me wish modules were more first class, so we could pass parameters and such, this is correct, the module doesn't know it's usage so it shouldn't try and figure it out.